pygmt/tests/test_xarray_accessor.py

```diff
@@ -14,10 +14,50 @@
 from pygmt.datasets import load_earth_relief
 from pygmt.enums import GridRegistration, GridType
 from pygmt.exceptions import GMTValueError
+from pygmt.helpers.testing import load_static_earth_relief
 
 _HAS_NETCDF4 = bool(importlib.util.find_spec("netCDF4"))
 
 
+@pytest.fixture(scope="module", name="grid")
+def fixture_grid():
+    """
+    Load the grid data from the sample earth_relief file.
+    """
+    return load_static_earth_relief()
+
+
+@pytest.fixture(scope="module", name="expected_clipped_grid")
+def fixture_expected_clipped_grid():
+    """
+    The expected grdclip grid result.
+    """
+    return xr.DataArray(
+        data=[
+            [1000.0, 570.5, -1000.0, -1000.0],
+            [1000.0, 1000.0, 571.5, 638.5],
+            [555.5, 556.0, 580.0, 1000.0],
+        ],
+        coords={"lon": [-52.5, -51.5, -50.5, -49.5], "lat": [-18.5, -17.5, -16.5]},
+        dims=["lat", "lon"],
+    )
+
+
+@pytest.fixture(scope="module", name="expected_equalized_grid")
+def fixture_expected_equalized_grid():
+    """
+    The expected grdhisteq grid result.
+    """
+    return xr.DataArray(
+        data=[[0, 0, 0, 1], [0, 0, 0, 1], [0, 0, 1, 1], [1, 1, 1, 1]],
+        coords={
+            "lon": [-51.5, -50.5, -49.5, -48.5],
+            "lat": [-21.5, -20.5, -19.5, -18.5],
+        },
+        dims=["lat", "lon"],
+    )
```

Comment:
These two fixtures are only used once. Probably ok to just put the `xr.DataArray` in the test itself?

Reply:
Done.